src/replica/replica_stub.cpp

```diff
@@ -1393,11 +1409,12 @@ void replica_stub::set_replica_state_subscriber_for_test(replica_state_subscribe
 // replica_stub::close
 // ThreadPool: THREAD_POOL_REPLICATION
 void replica_stub::on_node_query_reply_scatter(replica_stub_ptr this_,
-                                               const configuration_update_request &req)
+                                               const configuration_update_request &req,
+                                               split_status::type meta_split_status)
 {
     replica_ptr replica = get_replica(req.config.pid);
     if (replica != nullptr) {
-        replica->on_config_sync(req.info, req.config);
+        replica->on_config_sync(req.info, req.config, meta_split_status);
```

Comment:
It's ugly to particularly add an argument for one requirement. It makes the function responsibility obscure. I'd prefer adding `meta_split_status` to `configuration_update_request`, even it's ugly too.

Think about it, dozens of parameters in a function, if we keep doing that.

Reply:
Done